ui/src/data-services/models/occurrence-details.ts

```diff
@@ -29,6 +29,8 @@ export class OccurrenceDetails extends Occurrence {
 
     return {
       id,
+      captureId:
+        detection.capture !== undefined ? `${detection.capture}` : undefined,
```

Comment:
If you want to add capture id to occurrence detections using a different key or something, this is where we need to change how to access it @mihow